jsapp/js/components/submissions/table.es6

```diff
@@ -461,6 +465,10 @@ export class DataTable extends React.Component {
         case ADDITIONAL_SUBMISSION_PROPS._submitted_by:
             index = 'z92';
             break;
+        // set index for `background-audio` to the front of every column with `_`
+        case 'background-audio':
+            index = '_1';
```

Comment:
Wouldn't `_1` make it appear as first column? Columns with `_` (so `_id`, `_uuid`, etc.) use `z123`, and the other columns use numbers, so underscore would come first

Reply:
Not sure I understand what's wrong :sweat_smile:

Reply:
I changed the language in case it was bad wordplay

Reply:
I thought it wasn't supposed to be first, sorry :D